linera-views/tests/queueview_tests.rs

```diff
@@ -4,19 +4,22 @@
 use linera_views::{
     memory::create_memory_context,
     queue_view::QueueView,
+    test_utils,
     views::{CryptoHashRootView, CryptoHashView, RootView, View},
 };
-use rand::{Rng, SeedableRng};
+use rand::Rng as _;
 
 #[derive(CryptoHashRootView)]
 pub struct StateView<C> {
     pub queue: QueueView<C, u8>,
 }
 
+// XXX TODO this suffers from a hash collision — re-enable once fixed
```

Comment:
Just a reminder that we may want to have an issue for this `TODO`.

Reply:
This is #1692 :)